library/core/src/slice/mod.rs

```diff
@@ -2599,6 +2599,32 @@ impl<T> [T] {
         }
     }
 
+    /// Fills `self` with elements returned by calling a closure repeatedly.
+    ///
+    /// This method uses a closure to create new values. If you'd rather
+    /// [`Clone`] a given value, use [`fill`]. If you want to use the [`Default`]
```

Comment:
Looks like rustdoc is not happy with ``[`fill`]`` as link. Adding ``[`fill`]: #method.fill`` at the end works, but I'm not sure how to do this with the automatic intra-doc links.

Reply:
Hmm, I'll update it to use the explicit version then. Not sure either what the right syntax is for auto intra doc links. I assumed it'd just work as it appears to in other places, but it's falling here. Maybe because `fill` is behind a different feature gate? I'm unsure too, heh.